tests/phpunit/tests/admin/wpSiteHealth.php

```diff
@@ -499,4 +499,80 @@ public function data_object_cache_thresholds() {
 			array( 'alloptions_bytes', 1000 ),
 		);
 	}
+
+	/**
+	 * Tests get_test_autoloaded_options() when autoloaded options less than warning size.
+	 *
+	 * @ticket 61276
+	 *
+	 * @covers ::get_test_autoloaded_options()
+	 */
+	public function test_wp_autoloaded_options_test_no_warning(): void {
+		$expected_label  = esc_html__( 'Autoloaded options are acceptable' );
+		$expected_status = 'good';
+
+		$result = $this->instance->get_test_autoloaded_options();
+		$this->assertSame( $expected_label, $result['label'] );
+		$this->assertSame( $expected_status, $result['status'] );
+	}
+
+	/**
+	 * Tests get_test_autoloaded_options() when autoloaded options more than warning size.
+	 *
+	 * @ticket 61276
+	 *
+	 * @covers ::get_test_autoloaded_options()
+	 */
+	public function test_wp_autoloaded_options_test_warning(): void {
+		self::set_autoloaded_option( 800000 );
+
+		$expected_label  = esc_html__( 'Autoloaded options could affect performance' );
+		$expected_status = 'critical';
+
+		$result = $this->instance->get_test_autoloaded_options();
+		$this->assertSame( $expected_label, $result['label'] );
+		$this->assertSame( $expected_status, $result['status'] );
+	}
+
+	/**
+	 * Tests get_autoloaded_options_size().
+	 *
+	 * @ticket 61276
+	 *
+	 * @covers ::get_autoloaded_options_size()
+	 */
+	public function test_get_autoloaded_options_size(): void {
+		global $wpdb;
+
+		$autoload_values = wp_autoload_values_to_autoload();
+
+		$autoloaded_options_size = $wpdb->get_var(
+			$wpdb->prepare(
+				sprintf(
+					"SELECT SUM(LENGTH(option_value)) FROM $wpdb->options WHERE autoload IN (%s)",
+					implode( ',', array_fill( 0, count( $autoload_values ), '%s' ) )
+				),
+				$autoload_values
+			)
+		);
+		$this->assertEquals( $autoloaded_options_size, $this->instance->get_autoloaded_options_size() );
```

Comment:
Since this is comparing integers, `$this->assertSame()` can be used for more strict testing.

Reply:
Nice catch!